docs/content/en/docs/whatsnew/changelog.md

```diff
@@ -15,6 +15,20 @@ Please upgrade to `v0.15.3` if you are using Tinkerbell (Bare Metal) provider.
 * When upgrading to a new minor version, a new OS image must be created using the new image-builder cli.
 {{% /alert %}}
 
+## [v0.17.2](https://github.com/aws/eks-anywhere/releases/tag/v0.17.2)
+
+### Supported OS version details
+|              | vSphere | Bare Metal |  Nutanix | CloudStack | Snow  |
+|    :---:     |  :---:  |   :---:   |   :---:  |    :---:   | :---: |
+| Ubuntu       | 20.04	 | 20.04     |	20.04             | Not supported	     | 20.04 |  
+|              | 22.04   | 22.04     |  22.04             | Not supported      | Not supported   |
+| Bottlerocket | 1.13.1  | 1.13.1    |  Not supported     | Not supported	     | Not supported   |
+| RHEL         | 8.7	   | 8.7	     |  Not supported     | 8.7	               | Not supported   |
+
+
+### Fixed
+- TODO
```

Comment:
Added support for configuring healthchecks on `EtcdadmCluster`s using `etcdcluster.cluster.x-k8s.io/healthcheck-retries` annotation [aws/etcdadm-controller#44](https://github.com/aws/etcdadm-controller/pull/44).